### PR TITLE
Tables

### PR DESCRIPTION
**Pull request by SamWhite745**

### CODE REVIEW

**Review by natashabuckham — APPROVED** [review]
This all looks great!

**Review by pablisch — COMMENTED** [review]
Should DMs table be lowercase dms?
I think you are mixing cases, e.g. INSTER INTO dms in seeds.sql.
My inclination is to stick with all lowercase for table names

**Review by natashabuckham — CHANGES_REQUESTED** [review]
change table name for DMs to lower case

File: spec/create_tables.sql
Comment (natashabuckham) [inline spec/create_tables.sql:1]: Rename columns in blocked table to blocker/blockee

**Review by natashabuckham — CHANGES_REQUESTED** [review]
Rename columns in blocked table

**Review by natashabuckham — APPROVED** [review]
Fab :)

**Pull request merged**